awx_collection/tests/integration/targets/role/tasks/main.yml

```diff
@@ -74,6 +91,9 @@
     - name: Add Joe to the new project by ID
       role:
         user: "{{ username }}"
```

Comment:
Should we still use `user` (and `team`) in tests if they're now deprecated?

Reply:
I left them in as while they are depreciated they should still be tested, however I added additional tests for the users feature.